runtime/caml/callback.h

```diff
@@ -50,8 +53,6 @@ CAMLextern void caml_startup_pooled (char_os ** argv);
 CAMLextern value caml_startup_pooled_exn (char_os ** argv);
 CAMLextern void caml_shutdown (void);
 
-CAMLextern int caml_callback_depth;
```

Comment:
`caml_callback_depth` should also be removed at line 62 in `compatibility.h`.

Reply:
compatibility.h is planned to be entirely removed in 5.00: https://github.com/ocaml/ocaml/pull/10831#discussion_r775447012.